src/pages/SignUpPage/SignUpPage.jsx

```diff
@@ -34,22 +65,59 @@ const SignUpPage = () => {
             className='input-box'
             id='password1'
             placeholder='********'
+            value={signupUser.password}
+            onChange={(e) => {
+              setError("noerror");
+              setSignUpUser((prev) => ({ ...prev, password: e.target.value }));
+            }}
+            // onFocus={() => setError("noerror")}
+            required
           />
-          <label htmlFor='password2' className='label-font-size'>
-            Confirm Password
-          </label>
-          <input
-            type='password'
-            minLength='8'
-            className='input-box'
-            id='password2'
-            placeholder='********'
-          />
+
           <label>
-            <input type='checkbox' name='checkbox' className='checkbox-size' />I
-            accept all Terms and Condition
+            <input
+              type='checkbox'
+              name='checkbox'
+              className='checkbox-size'
+              defaultChecked
+            />
+            I accept all Terms and Condition
           </label>
-          <button className='button login-button'>Create New Account</button>
+          <button
+            type='submit'
+            className='button login-button'
+            onClick={(e) => {
+              e.preventDefault();
+              let validRegex =
+                /^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/;
+              if (
+                signupUser.firstName !== "" &&
```

Comment:
Instead of checking whether the input is empty or not just add required attribute